docs/docs/recipes.md

```diff
@@ -228,7 +228,48 @@ Data sourcing in Gatsby is plugin-driven; Source plugins fetch data from their s
 
 ## Transforming data
 
-Transforming data in Gatsby is also plugin-driven; Transformer plugins take data fetched using source plugins, and process it into something more usable (e.g. JSON into JavaScript objects, markdown to HTML, and more).
+Transforming data in Gatsby is plugin-driven. Transformer plugins take data fetched using source plugins, and process it into something more usable (e.g. JSON into JavaScript objects, markdown to HTML, and more). `gatsby-transformer-plugin` can transform your markdown files to HTML.
 
-- Walk through an example using the `gatsby-transformer-remark` plugin to transform markdown files [tutorial part six](/tutorial/part-six/#transformer-plugins)
-- Search available transformer plugins in the [Gatsby library](/plugins/?=transformer)
+### Prerequisites
+
+- A Gatsby site with `gatsby-config.js` and an `index.js` page
+- A Markdown file saved in your Gatsby site `src` directory
+- The `gatsby-transformer-remark` plugin installed
+
+### Directions
+
+1. Add the plugin in your `gatsby-config.js` page:
+
+```
+// In your gatsby-config.js
+plugins: [`gatsby-transformer-remark`],
+```
+
+2. Add a GraphQL query to the `index.js` file of your Gatsby site to fetch `MarkdownRemark` nodes:
+
+```jsx:title=src/pages/index.js
+export const query = graphql`
+  query {
+    allMarkdownRemark {
+      totalCount
+      edges {
+        node {
+          id
+          frontmatter {
+            title
+            date(formatString: "DD MMMM, YYYY")
+          }
+          excerpt
+        }
+      }
+    }
+  }
+`
+```
+
+3. Restart the development server and open GraphiQL at `http://localhost:8000/___graphql`. Explore the fields available on the `MarkdownRemark` node.
+
+### Related links
```

Comment:
```suggestion
### Additional resources
```
The format has evolved a bit, here's the new heading to use!

Reply:
Thanks for the update! Will keep checking the new format to stay in sync.